components/Slider/src/MDCSlider.m

```diff
@@ -329,10 +323,6 @@ - (void)thumbTrackTouchCanceled:(__unused MDCThumbTrack *)thumbTrack {
   [self sendActionsForControlEvents:UIControlEventTouchCancel];
 }
 
-+ (UIColor *)defaultColor {
```

Comment:
just an fyi: we may have set this up for subclass use. See as we are discouraging that im ok with removing it.

Reply:
Sure, but I didn't see any uses of this method, so removed. It is also not in the [documentation](https://github.com/material-components/material-components-ios/blob/develop/components/Slider/README.md).